Makefile

```diff
@@ -6,7 +6,7 @@ PWD=$(shell pwd)
 
 hooks:
 	mkdir -p alpm-hooks/usr/share/libalpm/hooks
-	find /usr/share/libalpm/hooks -exec ln -s /dev/null $(PWD)/alpm-hooks{} \;
+	find /usr/share/libalpm/hooks -exec ln -sf /dev/null $(PWD)/alpm-hooks{} \;
```

Comment:
Do we really need force here?

Reply:
The reason I added it is that the first run failed due to missing fakechroot. This exposed the fact that an aborted run would result in no cleanup of the symlinks. The -f fixes this potential issue that may be faced by anyone working on future changes to this package since it removes the errors that would be seen on a subsequent run after an aborted run. If you have strong objections to the force, feel free to remove it, but it resolves a headache I encountered while testing.